server/api/apiSpecification.yaml

```diff
@@ -0,0 +1,568 @@
+openapi: 3.0.0
+info:
+  description: >
+    _This document describes the REST API of Ark Automate._
+    Ark Automate is a platform that allows office users and software developers to automate business or everyday processes by simply sketching the steps of their process. By using simple flowcharts or powerful BPMN in their process outlines, users can create small software solutions using RPA that finish their tasks much faster and more reliably.
+  version: 1.0.0
+  title: Ark API
+  license:
+    name: LICENSE (MIT)
+    url: https://github.com/bptlab/ark_automate
+
+servers:
+  - url: '{protocol}://{server}/api/v1'
+    variables:
+      protocol:
+        default: https
+        enum: [http, https]
+      server:
+        default: localhost:5000
```

Comment:
Shouldn't this refer to our heroku server?

Reply:
localhost:5000 is refering to our heroku server because at the moment we run our backend locally on localhost:5000 (with the command heroku local).